frontend/src/hooks/useNavigation.ts

```diff
@@ -32,6 +32,11 @@ export function useNavigation(): [string, INavigation[]] {
     }
   }, [navigation, location, menu])
 
+
+  useEffect(() => {
+    console.log("LOCATION :", location)
+  }, [location])
+
```

Comment:
Maybe don't need this? Just test code?

Reply:
i is for other task `history task` was my bad commit